src/tyr/matrix_serializer.cc

```diff
@@ -2,7 +2,7 @@
 
 #include "baldr/json.h"
 #include "proto_conversions.h"
-#include "thor/costmatrix.h"
+#include "thor/matrix_common.h"
```

Comment:
I moved most common things, which were actually previously defined in `costmatrix.h` to `matrix_common.h` as the import pattern was a bit weird otherwise (timedistance.h matrix would import costmatrix.h..)